src/Resources/config/services.xml

```diff
@@ -13,9 +13,10 @@
             <parameter key="create_basket">/swplatform/extensionstore/baskets</parameter>
             <parameter key="order_basket">/swplatform/extensionstore/orders</parameter>
             <parameter key="payment_means">/swplatform/extensionstore/paymentmeans</parameter>
-            <parameter key="iaf_details">/swcloudadmin/store/extensions/%%d/inappfeatures/%%s</parameter>
-            <parameter key="iaf_create_basket">/swcloudadmin/inappfeatures/baskets</parameter>
-            <parameter key="iaf_order_basket">/swcloudadmin/inappfeatures/orders</parameter>
+            <parameter key="iap_details">/swcloudadmin/store/extensions/%%d/inappfeatures/%%s</parameter>
+            <parameter key="iap_create_basket">/swcloudadmin/inappfeatures/baskets</parameter>
+            <parameter key="iap_order_basket">/swcloudadmin/inappfeatures/orders</parameter>
+            <parameter key="iap_list">/swcloudadmin/store/extensions/%d/inappfeatures</parameter>
```

Comment:
this should not be `swcloudadmin`, which is for Rufus, this should be `swplatform`.

Reply:
if you look at the json provided in the ticket, it's `swcloudadmin`

![image](https://github.com/user-attachments/assets/d7adcd77-f4d2-4c01-b537-551bffd42739)

I don't make the rules, I just work here. 😂

Reply:
but that's why I told you yesterday, there are always two routes for each endpoint, one for SaaS and one for onPrem. Just because his demo showed SaaS, this is not the right choice for onPrem (which SwagExtensionStore is by default)